server/api/test/index.js

```diff
@@ -46,7 +47,7 @@ describe( 'api', function() {
 	} );
 
 	let maybeIt = it;
-	if ( process.env.NODE_ENV !== 'desktop' ) {
```

Comment:
It would need to be double checked in Automattic/wp-desktop repository. It might be working somehow there. I have no idea :)